tasks/databases.rake

```diff
@@ -90,10 +117,32 @@ namespace :db do
         migrations.each do |migration|
           if migration[:kind] == :data
             ActiveRecord::Migration.write("== %s %s" % ['Data', "=" * 71])
-            DataMigrate::DataMigrator.run(:up, "db/data/", migration[:version])
+            if time_spent_and_memory_pressure
+              print_memory_usage do
+                print_time_spent do
+                  DataMigrate::DataMigrator.run(:up, "db/data/", migration[:version])
+                  puts "The following statistics are for migration version: #{migration[:version]}, and direction: 'up'"
```

Comment:
I see similar code block in a few places, is it possible to move it into a reusable place?